server/routes.go

```diff
@@ -352,6 +420,7 @@ func Serve(ln net.Listener, extraOrigins []string) error {
 
 	r.POST("/api/pull", PullModelHandler)
 	r.POST("/api/generate", GenerateHandler)
+	r.POST("/api/embedding", EmbeddingHandler)
```

Comment:
Consider `embeddings` ?

```suggestion
	r.POST("/api/embeddings", EmbeddingHandler)
```